### DISCUSSION TITLE
Does github consider .mailmap file?

### DISCUSSION THREAD

**Discussion by huyvohcmc** [community/community#41583] [New to GitHub]
I am contributing to a private repo with wrong email author (typo), and want to correct it, so the contribution graph of the repo can show contributions linked to my account. Is it achievable with a .mailmap file?

**Comment by DSDmark** [comment]
Hi @huyvohcmc,

Did you try this, inside your settings

![image](https://user-images.githubusercontent.com/85744816/207245753-bdc26f6d-f59a-4139-a838-ae50b397af69.png)

https://github.com/settings/profile

**Reply by huyvohcmc** [reply]
I mean the contributors page in the repository (updated).

**Comment by radar-blip** [comment]
Some info here: https://stackoverflow.com/questions/53629125/does-github-consider-mailmap-for-contribution-graph
Looks like some had success, others not so much.
